server/src/main/java/io/whitefox/core/services/HadoopConfigBuilder.java

```diff
@@ -0,0 +1,38 @@
+package io.whitefox.core.services;
+
+import io.whitefox.core.AwsCredentials;
+import io.whitefox.core.Storage;
+import io.whitefox.core.StorageProperties;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.apache.hadoop.conf.Configuration;
+
+@ApplicationScoped
```

Comment:
Are you sure we need this? No one injects this

Reply:
You're right